tox.ini

```diff
@@ -164,26 +160,16 @@ changedir =
 
 commands_pre =
 ; Install without -e to test the actual installation
-  python -m pip install -U pip setuptools wheel
-
+  py3{4,5,6}: python -m pip install -U pip setuptools wheel
```

Comment:
Any reason why we still need this line if it is run in `travis.yml`?

Reply:
Unfortunately, upgrading pip in travis before bringing up virtualenvs via tox was part of a failed experiment. I've reverted the change in the travis file, and kept the pip install -U command for the versions of python that needed it.